Src/BootCamp.Chapter/Items/Weapon.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace BootCamp.Chapter.Items
+{
+    public class Weapon
+    {
+        private string _name;
+        private decimal _price;
```

Comment:
Codacy found an issue: [Remove this unread private field '_price' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)